### ISSUE TITLE
Yet another ttm:role issue - can script type values be used on script events?

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/dapt#83]
Considering how to mark up a document that's mid-stage in the workflow between two script types, for example an Original Dialogue List that's being translated, let's say half of it has been translated.

First question: what should the script type be?

Second question: Would it be reasonable to mark up each Script Event with a `ttm:role` indicating its contents, using the same values as we are defining for Script Type? For example those events that have been translated could be marked as `x-DUBBING_TRANSLATED_DIALOGUE_LIST` whereas those that have not yet been translated could be marked as `x-DUBBING_ORIGINAL_DIALOGUE_LIST`.

Since there's no restriction on the content elements that can have a `ttm:role` attribute we can't actually _stop_ people from doing this, so I think it would make sense to embrace it.

We might want to define how the `tt/head@ttm:role` attribute value should be set in case this practice is adopted. For example, "The Script type SHOULD indicate the earliest workflow step used by any of the Script Events in the document" or something like that.

**Comment by cconcolato** [comment]
I don't think DAPT script should represent intermediate states. I don't expect authoring tools to exchange half-finished documents.

**Comment by cconcolato** [comment]
This is related to #52

**Comment by cconcolato** [comment]
Because it might be tricky to define exact values in the spec with precise semantics for a script event type, we would prefer if the spec used a generic "string" type and define a registry where each company could define its own types.

**Comment by nigelmegitt** [comment]
A registry is fine. Although it might be tricky, I think it would be beneficial for industry in general at least to try to converge on commonly used values where possible, with an extension mechanism where none of the defined values works.

**Comment by cconcolato** [comment]
The feels almost as a duplicate of #140

**Comment by nigelmegitt** [comment]
Agree, #140 looks like a duplicate of this issue in many ways. Certainly we should be able to address both with the same decision.

**Comment by nigelmegitt** [comment]
Closing as duplicate of #140.
